server/validators/doesNotContainMarkup.ts

```diff
@@ -0,0 +1,18 @@
+import { registerDecorator, ValidationArguments, ValidationOptions } from 'class-validator'
+
+export default function DoesNotContainMarkup(validationOptions?: ValidationOptions) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const doesNotContainMarkup = (value: string, { object }: ValidationArguments) => {
+    return !(value.includes('<') || value.includes('>'))
```

Comment:
Not sure that disallowing characters is the way to go. We should just escape the content that the user has entered..

If we could find out why the markup is escaped in other fields but not this field, it might go a ways towards finding the real issue we have

Reply:
The way we concatenate the address with commas isn't good either, if the user enters "text1, test2" as a single line, it will get parsed as 2 lines because of the comma

Reply:
Agree with the address fields concatenated into the DB - all a bit awkward. I did initially escape the values, but found that this caused further issues in saving/reading values to/from the DB via the API - it depends how invalid characters are escaped. Also, tried limiting characters in a nunjucks `picture` - but clearer for the user when validated together. The flip-side is that there should never be < or > characters in addresses - and avoiding these getting into the system, with a clear validation message, does tell the user what is wrong with their input and why they aren't allowed.